api/models/version_endpoint.go

```diff
@@ -45,9 +45,9 @@ type VersionEndpoint struct {
 	// ServiceName service name
 	ServiceName string `json:"service_name" gorm:"service_name"`
 	// InferenceServiceName name of inference service
-	InferenceServiceName string `json:"-" gorm:"inference_service_name"`
+	InferenceServiceName string `json:"inference_service_name" gorm:"inference_service_name"`
 	// Namespace namespace where the version is deployed at
-	Namespace string `json:"-" gorm:"namespace"`
+	Namespace string `json:"namespace" gorm:"namespace"`
```

Comment:
why is the marshalling now when it previously wasn't needed

Reply:
Previously we fetched version endpoint value from the database, but now we will get the version endpoint from job queue which is stored in JSON and need to be unmarshal properly.